tests/test_fx.py

```diff
@@ -206,7 +207,15 @@ def test_speedx():
     clip2.write_videofile(os.path.join(TMP_DIR, "speedx3.webm"))
 
 def test_supersample():
-    pass
+    clip = VideoFileClip("media/big_buck_bunny_432_433.webm")
+
+    clip1 = supersample(clip, clip.duration, 1)
+    assert clip1.duration == clip.duration
+    clip1.write_videofile(os.path.join(TMP_DIR, "supersample1.webm"))
+
+    clip2 = supersample(clip, clip.duration / 2, 2)
+    assert clip2.duration == clip.duration
+    clip2.write_videofile(os.path.join(TMP_DIR, "supersample2.webm"))
```

Comment:
I don't know, maybe it is just me. But this test passes even without your changes. Which leads to question: what the reason of this PR.

I mean there is definitely a problem has been solve, but tests didn't reflect this fact. In other words there is no tests to reproduce an issue you have tried to solve.